app/views/coronavirus_landing_page/components/landing_page/_nhs_notice.html.erb

```diff
@@ -21,12 +21,18 @@
 
   <div class="govuk-grid-column-two-thirds">
     <% nhs_banner["sections"].each do |section| %>
-      <%= render "govuk_publishing_components/components/heading", {
-        text: section["heading"],
-        heading_level: 2,
-        font_size: "s",
-        margin_bottom: 4,
-      } if section["heading"].present? %>
+      <% if section["link_url"].present? %>
+        <p class="gem-c-heading govuk-heading-s govuk-!-margin-bottom-4">
```

Comment:
The `gem-c-heading` class can probably be removed here as it's a component specific class - `govuk-heading-s` should be sufficient. Also we tend to avoid using the margin override classes if possible unless they're actually overriding something. It might work if you tried `govuk-body` instead?

Reply:
Thanks for the suggestions, will try them out and fixup.  Also, fix the breaking test(s)...

Reply:
Thanks @andysellick - styles updated leaving only `govuk-heading-s` - visually and margin/padding wise looks the same.